### DISCUSSION TITLE
Code Table Admin Meeting Reminder

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6306] [Announcements]
@ArctosDB/arctos-code-table-administrators 

Just a reminder that we have a meeting scheduled for tomorrow, Thursday, May 18 at 1PM Pacific Time. I will probably miss this meeting as I have a conflict. I would very much like to get the attributes requested by NCWRC approved as this is holding up incoming data. I have placed those issues on the [agenda](https://docs.google.com/document/d/1eZ_S8VTp-m5Wrbb6cAK96ODPZpt2-Hvnvt6yfU9qVwg/edit?usp=sharing).

However, there is a [HUGE backlog of code table requests](https://github.com/orgs/ArctosDB/projects/23/views/1). We cannot address code table requests only at monthly meetings. We need to be reviewing these in a more timely fashion, or we need a new process for approving new code table terms. Ideas are appreciated.

I will try to make the meeting, if I can.

Adios,
Teresa

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I am also unfortunately unable to make the meeting this month.

**Comment by ccicero** [comment]
I will hop on to see if anyone else is on the call.

**Comment by campmlc** [comment]
I'll try to make it.


Mariel Campbell   (she/her)
Collection Manager, Division of Genomic Resources
Museum of Southwestern  Biology

Shipping Address:
Museum of Southwestern Biology
Division of Genomic Resources
1 University of New Mexico
MSC03 2020
Albuquerque, NM 87131

(505) 277-7808 office
email: ***@***.***


________________________________

**Comment by dustymc** [comment]
I will be there.

I think we need some sort of ctadmin howto doc. I think many of the open issues (eg, antler stuff) are noncontroversial, ctadmins should read them and either leave comments or check boxes, and then the meetings can be reserved for addressing the comments. (Or some of them, a few things probably need full AWG attention - do we need some flag for that?) Is that reasonable? If so, can we make it happen? If not, is https://github.com/ArctosDB/arctos/issues/new?assignees=&labels=Function-CodeTables&projects=&template=code-table-request.md&title=Code+Table+Request+-+ itself unreasonable? I suppose I'm up for most anything - other than not addressing user's needs, which is where we seem to be now, or creating denormalized and undocumented messes, which is what lead to where we are now....

**Comment by ebraker** [comment]
I will be there!
